components/Link/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import { Link } from "../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Navigation/Link"
+  component={Link}
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Usage
+
+This component can be used in two ways:
+
+- As a standard `a`-tag with icon support. For this you need to supply at least an `href`, and an icon can be added in the `icon` prop:
+  `<Link href="/home" icon={<ArrowRightIcon />} iconAlign="end">Home</Link>`
+
+- With another way of routing, for example using `react-router-dom`. For this we have exposed the `component` property. When adding a `component` all props of this component will be usable on our `Link` component and will be passed through.
+  An example with `react-router-dom`:
+
+```
+import { Link } from '@gemeente-denhaag/denhaag-component-library';
+import { ArrowRightIcon } from '@gemeente-denhaag/icons';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link as RouterLink,
+} from 'react-router-dom';
+
+export default function App() {
+  return (
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li>
+               <Link component={RouterLink} to="/" icon={<ArrowRightIcon />}>Home</Link>
+            </li>
+            <li>
+               <Link component={RouterLink} to="/nl" hrefLang="nl">NL</Link>
+            </li>
+          </ul>
+        </nav>
+         <Switch>
+            <Route path="/nl">
+              <h1>NL home</h1>
+            </Route>
+            <Route path="/">
+              <h1>Home</h1>
+            </Route>
+          </Switch>
+      </div>
+    </Router>
+  );
+}
+```
```

Comment:
This documentation block is very React specific, should probably just be removed from the HTML story.

Reply:
Just removed it, think we looked over this react code in the bem stories when we did a git rebase

Reply:
Nice!